interfaces/builtin/common.go

```diff
@@ -142,3 +145,15 @@ func (iface *commonInterface) SecCompConnectedPlug(spec *seccomp.Specification,
 	}
 	return nil
 }
+
+func (iface *commonInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.Plug, plugAttrs map[string]interface{}, slot *interfaces.Slot, slotAttrs map[string]interface{}) error {
+	old := "###SLOT_SECURITY_TAGS###"
```

Comment:
I prefer to use the following code for a few reasons here, instead of introducing tags and doing the string.Replace here.
1. less code for consistency with other functions in commonInterface 
2. In favor of "format" here rather than "replace" for performance consideration (nitpicking, we have a large code snippet udev rules for udisk2,  modemManager, maybe more in the future)

``` 
if iface.connectedPlugUdev != "" {
        for appName := range plug.Apps {
            tag := udevSnapSecurityName(plug.Snap.Name(), appName)
            spec.AddSnippet(fmt.Sprintf(iface.connectedPlugUdev, tag))
        } 
}
```
People could use %[1]s if there're multiple udev entries there or simply use "%s" if only one entry exists.

 I'll leave the decision to you though.